styles/wiki.nixos.org/catppuccin.user.css

```diff
@@ -57,33 +59,12 @@
     @mantle: @catppuccin[@@lookup][@mantle];
     @crust: @catppuccin[@@lookup][@crust];
     @accent-color: @catppuccin[@@lookup][@@accent];
-
-    --ctp-rosewater: @rosewater;
-    --ctp-flamingo: @flamingo;
-    --ctp-pink: @pink;
-    --ctp-mauve: @mauve;
-    --ctp-red: @red;
-    --ctp-maroon: @maroon;
-    --ctp-peach: @peach;
-    --ctp-yellow: @yellow;
-    --ctp-green: @green;
-    --ctp-teal: @teal;
-    --ctp-sky: @sky;
-    --ctp-sapphire: @sapphire;
-    --ctp-blue: @blue;
-    --ctp-lavender: @lavender;
-    --ctp-text: @text;
-    --ctp-subtext1: @subtext1;
-    --ctp-subtext0: @subtext0;
-    --ctp-overlay2: @overlay2;
-    --ctp-overlay1: @overlay1;
-    --ctp-overlay0: @overlay0;
-    --ctp-surface2: @surface2;
-    --ctp-surface1: @surface1;
-    --ctp-surface0: @surface0;
-    --ctp-base: @base;
-    --ctp-mantle: @mantle;
-    --ctp-crust: @crust;
```

Comment:
These variables are key to adding syntax highlighting, see https://github.com/catppuccin/userstyles/blob/main/docs/tips-and-tricks.md#pygments.